enterprise/frontend/src/embedding-sdk/store/auth.ts

```diff
@@ -43,6 +46,23 @@ export const initAuth = createAsyncThunk(
       dispatch(refreshSiteSettings({})),
     ]);
 
+    const mbVersion = siteSettings.payload?.version?.tag;
+    const sdkVersion = getEmbeddingSdkVersion();
+
+    if (mbVersion && sdkVersion !== "unknown") {
+      if (
+        !isSdkVersionCompatibleWithMetabaseVersion({
+          mbVersion,
+          sdkVersion,
+        })
+      ) {
+        console.warn(
+          ...bigWarningHeader("Detected SDK compatibility issue"),
+          `SDK version ${sdkVersion} is not compatible with MB version ${mbVersion}, this might cause issues.`,
```

Comment:
Can we do more for our users? I would want to know what version we would recommend. Something like this would make me appreciate our SDK more.

"SDK version 1.51.4 is not compatible with MB version 0.50.1, please use MB version 1.51.x instead."

Reply:
I actually started writing the code and test for this... but then I realized that we can't know for sure which version is wrong... they may be using the latest sdk but pointing it to an outdated version of mb, or they be using a newer/old version of the sdk.
I'll try to come up with something